xml/selinux.xml

```diff
@@ -282,7 +259,8 @@ system_u:object_r:var_t var</screen>
 
   <para>
    The most important line in the security context is the context type. This
-   is the part of the security context that ends in _t. It tells &selnx;
+   is the part of the security context that ends in <literal>_t</literal>. 
```

Comment:
is the part of the security context that *often* ends in

Reply:
:+1: